File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
#include<bits/stdc++.h>
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        vector<int> result;
        unordered_map<int,int> mpp;
        for(auto x : nums){
            mpp[x]++;
        }
        int n = nums.size();
        while(k>0 && n>=1){
            for(auto x : nums){
                if(mpp[x] == n && find(result.begin(), 
                 result.end(), x) == result.end()){
                    result.emplace_back(x);
                    k--;
                }
            }
            n--;
        }
        return result;
    }
};

File: Data Structures & Algorithms/non-cyclical-number/submission-0.py
class Solution:
    def isHappy(self, n: int) -> bool:
        mpp = {}
        while (mpp.get(n) == None):
            mpp[n] = 1
            x = 0
            while (n > 0):
                x = x + int((n%10)**2)
                n = int(n/10)
            
            if x == 1:
                return True
            n = x
        return False
        